.circleci/config.yml

```diff
@@ -1317,6 +1317,9 @@ workflows:
       # Final reports
       - coverage_report: *requires_tests
 
+  test_latest:
+    <<: *workflow_test
+
```

Comment:
Can this latest/unpinned run be part of `test_nightly`?

Reply:
It can't be a part of test_nightly (it's another workflow), but it can be launched at the same time.
Do we want it at the same time of `test`, `test_nightly` or both?

Reply:
is it ok for you like that?

Reply:
Non-blocking: we should consider whether `test_latest` should have triggers and how failures are handled (both in terms of GitHub merges but also notifications to the team for remediation).